lib/run_loop/sim_keyboard_settings.rb

```diff
@@ -1,5 +1,14 @@
 module RunLoop
   class SimKeyboardSettings
+    PLIST_KEYS = {
+      hw_keyboard_seen: 'HardwareKeyboardLastSeen',
+      soft_keyboard_shown: 'SoftwareKeyboardShownByTouch',
+      minimization_enabled: 'AutomaticMinimizationEnabled',
+      keyboard_autocorrection: 'KeyboardAutocorrection',
+      keyboard_caps_lock: 'KeyboardCapsLock',
+      keyboard_autocapitalization: 'KeyboardAutocapitalization'
+    }.freeze
```

Comment:
I would caution against using constant keys.

When you are debugging this later, you might find yourself having to toggle between 3 places - this list, the usage of the constant in the code, and the actual plist.

I would prefer using the actual key name in the usage.

This is a non-blocking comment; just a suggestion.

Reply:
yeah, I think you're right. I wanted to get rid of redundancy of using these keys in several places, but got other problems introducing constants :)